### ISSUE TITLE
Contact Us accordions are not contained within the same group

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12087]

**Comment by davidmpickett** [comment]
Removing this from Lovell as it is a VAMC-wide issue

**Comment by kmariepat-cityfriends** [comment]
Next Steps:
- @kmariepat-cityfriends look into KB articles and take this to VHA DM sync for awareness and to communicate with editors

**Comment by davidmpickett** [comment]
@mmiddaugh Add this to discuss the next VHA DM sync

**Comment by mmiddaugh** [comment]
For clarity/future conversations:

- The alpha accordions on the Contact us page for Pittsburgh were built incorrectly using _three_ accordion `groups`, rather than by adding accordion `items` to a _single_ accordion `group`. This error was then cloned in the creation of all subsequent Contact us pages. 
   ->  it doesn't seem fair to make all editors address a problem that they did not cause and which won't be a teachable moment for them. 
- It is a level 4 defect which is annoying but does not prevent Veterans or beneficiaries from accessing the information that they need. 
- Fixing this specific problem is not a quick or easy fix. 

Bottom line: The Contact us page is a strong candidate for hardening at some point and this problem should be addressed as part of that effort as fixing it independently is more of an effort than it merits.

**Comment by CherylEvans** [comment]
This ticket was recreated in va.gov-team Ticket # [64841](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/64841)
